ANSWERS.md

```diff
@@ -0,0 +1,37 @@
+1) 
+What are PropTypes used for? Please describe why it's important to type check our data in JavaScript.
+    Prop types are used to export a range of validators that can be used to make sure the data you receive is valid.
+
+    As your app grows, you can catch a lot of bugs with typechecking. 
+
+    When an invalid value is provided for a prop, a warning will be shown in the JavaScript console. For performance reasons, propTypes is only checked in development mode.
+
+2)
+Describe a life-cycle event in React?
+
+    An event has three phases to its life cycle : cycle , target and bubbling
```

Comment:
Check out info on react life cycle methods I.e. Component did mount, etc for an in depth look at this question